src/styles/page-template/index.md.njk

```diff
@@ -41,13 +41,13 @@ You can customise the page template, for example, by adding a service name and n
 
 ### Configuring Nunjucks
 
-When using Nunjucks you’ll need to add `node_modules/govuk-frontend/` and `node_modules/govuk-frontend/components/`
+When using Nunjucks you’ll need to add `node_modules/govuk-frontend/` and `node_modules/govuk-frontend/govuk/components/`
```

Comment:
This isnt what we want but I think will get covered when we do the imports card...